packages/@okta/vuepress-site/docs/guides/pwd-optional-widget-sign-in-email/main/aspnet/integrationsteps.md

```diff
@@ -0,0 +1,58 @@
+The following code defines an endpoint that retrieves the current Sign-In Widget configuration details from session state and appends the `OTP` and `state` values from the callback to it. If it cannot find the configuration details in session state, it assumes the callback has not come from the same browser and prompts the user accordingly.
```

Comment:
Agreeing with Chris with the slight awkwardness of the sentence. A possible alternative is 👍 

"The following code defines an endpoint that receives the OTP and state values from the magic link. The code assigns these OTP and state values to the Sign-in Widget configuration object stored in session."

Reply:
Changed to "The following code defines an endpoint that receives the `OTP` and `state` values from the magic link. It then attempts to retrieve the current Sign-In Widget configuration details from session state. If successful, the `OTP` and `state` values are added to the configuration details and passed to the Widget. If unsuccessful, the code assumes the callback has not come from the same browser and prompts the user accordingly."